.crawler/src/constants.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2019-2022 Aleo Systems Inc.
+// This file is part of the snarkOS library.
+
+// The snarkOS library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The snarkOS library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the snarkOS library. If not, see <https://www.gnu.org/licenses/>.
+
+use snarkos_environment::{Client, CurrentNetwork, Environment};
+
+pub const DESIRED_CONNECTIONS: usize = <Client<CurrentNetwork>>::MINIMUM_NUMBER_OF_PEERS * 3;
+pub const LOG_INTERVAL_SECS: u64 = 10;
+pub const MAXIMUM_NUMBER_OF_PEERS: usize = 10000;
```

Comment:
We can probably handle that many at a time, but perhaps we should decrease it (to maybe 1k?) so as not to run into open file descriptor issues.